content/en/sensitive_data_scanner/setup/cloud_storage.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
This page does not need to be reviewed. Getting reviewed in child PR: https://github.com/DataDog/documentation/pull/27589

Reply:
Merged in.